src/Jenssegers/Mongodb/Connection.php

```diff
@@ -38,7 +38,7 @@ public function __construct(array $config)
         $this->connection = $this->createConnection($dsn, $config, $options);
 
         // Select database
-        $this->db = $this->connection->selectDatabase($config['database']);
+        $this->db = $this->connection->selectDatabase($this->getDatabaseDsn($dsn, $config['database']));
```

Comment:
When I don't set `$config['database']`, I get this error:
`Undefined index: database {"userId":7,"exception":"[object] (ErrorException(code: 0): Undefined index: database at /var/www/device-service/vendor/jenssegers/mongodb/src/Jenssegers/Mongodb/Connection.php:41)`

I think you should pass `@$config['database']` here.